[2022-03-30 19:59:35,215] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: IngestToGCP.processing_tasks_users.ingest_users_to_bucket scheduled__2008-08-01T00:00:00+00:00 [queued]>
[2022-03-30 19:59:35,346] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: IngestToGCP.processing_tasks_users.ingest_users_to_bucket scheduled__2008-08-01T00:00:00+00:00 [queued]>
[2022-03-30 19:59:35,346] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-30 19:59:35,346] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-30 19:59:35,346] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-30 19:59:35,454] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): processing_tasks_users.ingest_users_to_bucket> on 2008-08-01 00:00:00+00:00
[2022-03-30 19:59:35,481] {standard_task_runner.py:52} INFO - Started process 3054 to run task
[2022-03-30 19:59:35,563] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'IngestToGCP', 'processing_tasks_users.ingest_users_to_bucket', 'scheduled__2008-08-01T00:00:00+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/ingest_dag.py', '--cfg-path', '/tmp/tmpn3dwdv9p', '--error-file', '/tmp/tmpnj3czfe3']
[2022-03-30 19:59:35,564] {standard_task_runner.py:77} INFO - Job 164: Subtask processing_tasks_users.ingest_users_to_bucket
[2022-03-30 19:59:36,009] {logging_mixin.py:109} INFO - Running <TaskInstance: IngestToGCP.processing_tasks_users.ingest_users_to_bucket scheduled__2008-08-01T00:00:00+00:00 [running]> on host d9825fd5ae59
[2022-03-30 19:59:36,257] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-03-30 19:59:36,402] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=IngestToGCP
AIRFLOW_CTX_TASK_ID=processing_tasks_users.ingest_users_to_bucket
AIRFLOW_CTX_EXECUTION_DATE=2008-08-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2008-08-01T00:00:00+00:00
[2022-03-30 19:59:38,573] {python.py:175} INFO - Done. Returned value was: None
[2022-03-30 19:59:38,668] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=IngestToGCP, task_id=processing_tasks_users.ingest_users_to_bucket, execution_date=20080801T000000, start_date=20220330T195935, end_date=20220330T195938
[2022-03-30 19:59:38,843] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-30 19:59:40,648] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-05 22:26:28,423] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: IngestToGCP.processing_tasks_users.ingest_users_to_bucket scheduled__2008-08-01T00:00:00+00:00 [queued]>
[2022-04-05 22:26:28,474] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: IngestToGCP.processing_tasks_users.ingest_users_to_bucket scheduled__2008-08-01T00:00:00+00:00 [queued]>
[2022-04-05 22:26:28,474] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-05 22:26:28,475] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-05 22:26:28,475] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-05 22:26:28,528] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): processing_tasks_users.ingest_users_to_bucket> on 2008-08-01 00:00:00+00:00
[2022-04-05 22:26:28,554] {standard_task_runner.py:52} INFO - Started process 512 to run task
[2022-04-05 22:26:28,574] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'IngestToGCP', 'processing_tasks_users.ingest_users_to_bucket', 'scheduled__2008-08-01T00:00:00+00:00', '--job-id', '8345', '--raw', '--subdir', 'DAGS_FOLDER/ingest_dag.py', '--cfg-path', '/tmp/tmpsfjzjipg', '--error-file', '/tmp/tmpaogds2n_']
[2022-04-05 22:26:28,575] {standard_task_runner.py:80} INFO - Job 8345: Subtask processing_tasks_users.ingest_users_to_bucket
[2022-04-05 22:26:28,730] {logging_mixin.py:109} INFO - Running <TaskInstance: IngestToGCP.processing_tasks_users.ingest_users_to_bucket scheduled__2008-08-01T00:00:00+00:00 [running]> on host 9fe4286b2855
[2022-04-05 22:26:28,959] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-05 22:26:29,054] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=IngestToGCP
AIRFLOW_CTX_TASK_ID=processing_tasks_users.ingest_users_to_bucket
AIRFLOW_CTX_EXECUTION_DATE=2008-08-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2008-08-01T00:00:00+00:00
[2022-04-05 22:26:29,305] {python.py:175} INFO - Done. Returned value was: None
[2022-04-05 22:26:29,549] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=IngestToGCP, task_id=processing_tasks_users.ingest_users_to_bucket, execution_date=20080801T000000, start_date=20220405T222628, end_date=20220405T222629
[2022-04-05 22:26:29,673] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-05 22:26:29,794] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
